modules/fts/pages/fts-rebalance-failover.adoc

```diff
@@ -0,0 +1,52 @@
+= Rebalance/Failover for Search Service
+
+== Rebalance 
```

Comment:
Did anyone from the docs team verified the language aspects of these contents? If not, please do for this PR.